src/vm/autotrace.cpp

```diff
@@ -0,0 +1,101 @@
+#include "common.h" // Required for pre-compiled header
+
+#ifdef FEATURE_AUTO_TRACE
+#ifdef FEATURE_PAL
+#include "pal.h"
+#endif // FEATURE_PAL
+
+HANDLE auto_trace_event;
+static size_t g_n_tracers = 1;
+#ifdef __apple__
+static const WCHAR* command_format = L"%hs -p %d";
+#else
+static const WCHAR* command_format = u"%hs -p %d";
+#endif // __apple__
+static WCHAR* command = nullptr;
+
+void auto_trace_init()
+{
+    char *nAutoTracersValue = getenv("N_AUTO_TRACERS");
```

Comment:
nit: Should we make these environment variables follow the same standard as the COMPlus_ prefixed environment variables?